miden-lib/asm/miden/note.masm

```diff
@@ -252,19 +255,3 @@ export.get_script_hash
     swapdw dropw dropw swapw dropw
     # => [SCRIPT_HASH]
 end
-
-# PROCEDURES COPIED FROM KERNEL (TODO: get rid of this duplication)
-# =================================================================================================
-
-# The maximum number of input values associated with a single note.
-const.MAX_INPUTS_PER_NOTE=128
-
-#! Returns the max allowed number of input values per note.
-#!
-#! Stack: []
-#! Output: [max_inputs_per_note]
-#!
-#! - max_inputs_per_note is the max inputs per note.
-export.get_max_inputs_per_note
```

Comment:
Is there a reason not to re-export the same procedure from here? For example, like:
```
export.utils::note::get_max_inputs_per_note
```